deployment/upgradeLive

```diff
@@ -120,10 +120,13 @@ function downgrade() {
 }
 
 function stop() {
+   echo "stopping liveController..."
```

Comment:
We should check for -x /etc/init.d/liveRecorder not -r, also, since we're checking, we should check for both.
Why do we echo "stopping ..." here? the "stopping" output should be done in the control scripts themselves, not here.